apps/tasks/views.py

```diff
@@ -0,0 +1,94 @@
+from rest_framework import filters
+from rest_framework import status, viewsets
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from django.contrib.auth.models import User
+from django.core.mail import send_mail
+from django.conf import settings
+from rest_framework.generics import get_object_or_404
+
+from apps.tasks.models import Task, Comment, TimeLog
+from apps.tasks.serializers import SerializerTask, TaskListSerializer, TaskDetailsByIdSerializer, AssignTask, \
+    CreateCommentSerializer, AllCommentSerializer, TimeLogSerializer
+
+
+def send_task_assignment_notification(task, subject, message):
```

Comment:
mai bine faci functia aceasta globala ca sa o poti utiliza in mai multe lucuri din proiect, faci un fisier in common/helpers.py si acolo pui functia acesta sub numele send_notification(...)